x/gamm/types/msgs_test.go

```diff
@@ -867,3 +869,98 @@ func TestMsgExitSwapShareAmountIn(t *testing.T) {
 		}
 	}
 }
+
+// Test authz serialize and de-serializes for gamm msg.
+func TestAuthzMsg(t *testing.T) {
+	appParams.SetAddressPrefixes()
```

Comment:
I wasn't sure why we need this line so I tried removing it and the test still passes
```suggestion
```